modules/hosted-control-planes-version-support.adoc

```diff
@@ -8,11 +8,32 @@
 [id="hosted-control-planes-version-support_{context}"]
 = Versioning for {hcp}
 
+The {hcp} feature includes the following components, which might require independent versioning an support levels:
+
+* Management cluster
+* HyperShift Operator
+* {hcp-capital} (`hcp`) command-line interface (CLI)
+* `hypershift.openshift.io` API
+* Control Plane Operator
+
+[id="hcp-versioning-mgmt_{context}"]
+== Management cluster
+
+In management clusters for production use, you need {mce}, which is available through OperatorHub. The {mce-short} bundles a supported build of the HyperShift Operator. For your management clusters to remain supported, you must use version of {product-title} that {mce-short} runs on. In general, a new release of {mce-short} runs on the following versions of {product-title}:
+
+* The latest Generally Available version of {product-title}
+* Two versions before the latest Generally Available version of {product-title}
```

Comment:
I don't see that we have explicit guidelines related to this anywhere, but, I'm thinking about whether this should be lowercase "generally available" (as a sort of general reference) vs "General Availability" (which is what I have typically known "GA" to specifically stand for, as a "Common Release Term" / release level).

I am sure I am just splitting hairs here. 😅 And FWIW, we definitely are not consistent on this across the repo (can find examples backing up either). 🤷 

So, just a suggestion:
```suggestion
* The latest General Availability version of {product-title}
* Two versions before the latest General Availability version of {product-title}
```

Reply:
I think you are right -- GA means General Availability. I'll make those changes.